client-contract/src/DealClient.sol

```diff
@@ -190,9 +190,10 @@ contract DealClient {
         ret.client = getDelegatedAddress(address(this));
```

Comment:
getDelegatedAddress has been fixed in the newer lib version. Remove the definition in this file and use the one available in the lib: https://github.com/Zondax/filecoin-solidity/blob/master/contracts/v0.8/utils/FilAddresses.sol#L35

Reply:
https://github.com/Zondax/filecoin-solidity/blob/d07293f079f502a5270edd002f15b470120b23f6/contracts/v0.8/utils/FilAddresses.sol#L35 

using this one as it has the "040" same as the old `getDelegatedAddress` fn